lib/Data.js

```diff
@@ -34,21 +36,17 @@ export class Suite {
     return this.tests.concat(childSuiteTests)
   }
 
-  get runtime () {
-    var status = this.status
-
-    if (status === 'skipped' || status === undefined) {
+  getRuntime () {
```

Comment:
Fixed in https://github.com/js-reporters/js-reporters/commit/a83b2534bba5b27dc3d368c9aae06e1802bea44b, sorry for this.